lib/saluki-components/src/sources/otlp/mod.rs

```diff
@@ -224,14 +267,15 @@ impl SourceBuilder for OtlpConfiguration {
         let translator_config = metrics::config::OtlpTranslatorConfig::default().with_remapping(true);
         let grpc_max_recv_msg_size_bytes = self.receiver.protocols.grpc.max_recv_msg_size_mib as usize * 1024 * 1024;
         let metrics = build_metrics(&context);
-
+        let logs = build_logs(); // TODO: finish implementing
```

Comment:
What is `logs` supposed to be here? `metrics` here isn't referring to any OTLP metric / DD metric. Its for telemetry, so you can add your log specific telemetry to the `Metrics` struct